Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -305,3 +310,10 @@ extension PartialIso where A == (String?, Int?), B == Pricing {
     })
   }
 }
+
+private let isTest: Router<Bool?> =
+  formField("live", .string).map(isPresent >>> negate >>> Optional.iso.some)
+    <|> formField("test", .string).map(isPresent >>> Optional.iso.some)
+
+private let isPresent = PartialIso<String, Bool>(apply: { _ in true }, unapply: { $0 ? "" : nil })
+private let negate = PartialIso<Bool, Bool>(apply: { !$0 }, unapply: { !$0 })
```

Comment:
and this stuff down here ain't great either, but maybe this is just how it has to be. I wanna parse out the `live` and `test` form fields, and map their presence to either `true` or `false`.